test/package.json

```diff
@@ -17,8 +17,8 @@
     "bedrock-express": "^3.2.0",
     "bedrock-https-agent": "^2.0.0",
     "bedrock-jsonld-document-loader": "^1.0.1",
-    "bedrock-kms": "^3.0.1",
-    "bedrock-kms-http": "^4.0.0",
+    "bedrock-kms": "^5.0.0",
+    "bedrock-kms-http": "digitalbazaar/bedrock-kms-http",
```

Comment:
Should this be published to a package, or is there ongoing work?

Reply:
Work is ongoing in `bedrock-kms-http`.  There will be a release made there before this is merged.

Reply:
@mattcollier np, thanks.